index.js

```diff
@@ -1064,14 +1126,13 @@ fileType.stream = readableStream => new Promise((resolve, reject) => {
 	});
 });
 
-Object.defineProperty(fileType, 'extensions', {
-	get() {
-		return new Set(supported.extensions);
-	}
-});
-
-Object.defineProperty(fileType, 'mimeTypes', {
-	get() {
-		return new Set(supported.mimeTypes);
-	}
-});
```

Comment:
These were using `Object.defineProperty` for a reason. They should not be changeable.

Reply:
Sorry, I am not familiar with that pattern, I will revert.